samza-core/src/main/java/org/apache/samza/container/grouper/task/TaskAssignmentManager.java

```diff
@@ -131,12 +141,13 @@ public void writeTaskContainerMapping(String taskName, String containerId) {
    */
   public void deleteTaskContainerMappings(Iterable<String> taskNames) {
```

Comment:
Do we need to delete task mode mapping wherever this is being called?

In general, would it make sense to get/put/delete them together as a complex value?